packages/@groupby/elements-search-driver-plugin/test/unit/common/search-driver-plugin.test.ts

```diff
@@ -86,20 +91,29 @@ describe('SearchDriverPlugin', () => {
   describe('fetchSearchData()', () => {
     const area = 'area';
     const collection = 'collection';
-    const query = 'search term';
     let config;
     let group;
-    let results;
+    let response;
     let sendSearchApiRequest;
+    let dispatchSearchBeacon;
+    let core;
+    let query;
 
     beforeEach(() => {
       config = { area, collection };
+      query = 'search term';
```

Comment:
Unless i missed something, this should be fine to put back as a `const` outside of the `beforeEach()`.